egs2/chime7_task1/asr1/run.sh

```diff
@@ -67,7 +67,7 @@ nbpe=500
 asr_max_epochs=8
 # put popcornell/chime7_task1_asr1_baseline if you want to test with pretrained model
 use_pretrained=
-decode_only=0
+decode_only=
```

Comment:
I'm concerned that when it is empty, line 206 `data_opts+=" --decode-only $decode_only"` will raise some error, e.g. `Error: No positional arguments are required`.

Reply:
now it is an empty string.